docs/content/en/_index.html

```diff
@@ -430,9 +430,9 @@ <h3 class="tools__item-title">Tetragon lab</h3>
                     </a>
                 </li>
                 <li class="tools__item">
-                    <a class="tools__link" href="/docs/getting-started/">
+                    <a class="tools__link" href="https://labs.iximiuz.com/tutorials/introduction-to-tetragon" target="_blank" rel="noopener noreferrer">
                         <img src="/svgs/promo-astronomy.svg" width="64" height="64" loading="lazy" alt="telescope"/>
-                        <h3 class="tools__item-title">Exploring Tetragon</h3>
+                        <h3 class="tools__item-title">Exploring Tetragon Lab</h3>
```

Comment:
nit, we used "Tetragon lab" on the other. Let's decide if we use capital letters or not but maybe not mix them

<img width="572" alt="image" src="https://github.com/cilium/tetragon/assets/11256051/1cd9ee15-324a-440f-bae4-ef80a014b485">